File: Sources/gestion-culturelle-back/src/main/java/com/afpa/cda/dto/ReservationDto.java
package com.afpa.cda.dto;

import java.util.Date;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class ReservationDto {

	private int id;
	
	private ManifestationDto manifestation;
	private UserDto client;
	
	String numClient;
	int quantite;
	Date dateReservation;
	
	double total;
	
}

File: Sources/gestion-culturelle-front/project/src/app/model/reservationDto.ts
import { ManifestationDto } from './manifestationDto';
import { User } from './user';


export class ReservationDto {
    id: number;
    manifestation: ManifestationDto;
    client:User;
    numClient:String;
    dateReservation:Date;
    total:number;


    constructor(id?: number,
        manifestation?: ManifestationDto,
        client?:User,
        numClient?:String,
        dateReservation?:Date,
        total?:number) {
        this.id = id;
        this.manifestation = manifestation;
        this.client=client;
        this.numClient=numClient;
        this.dateReservation=dateReservation;
        this.total=total

    }
}


